frontend/index.html

```diff
@@ -7,38 +7,33 @@
 
         <title>contained.af</title>
 
-        <link href="css/contained.min.css" media="all" rel="stylesheet" />
     </head>
     <body>
-        <div class="container">
-            <div class="page-header">
-                <h1>Container Escape Bounty CTF</h1>
-            </div>
+        <h1>Profiles</h1>
+        Provide details here and you will be redirected to the correct page.
+        <br><br>
 
-
-            <div id="console"></div>
-
-            <footer class="footer">
-                <div class="row">
-                    <div class="col-md-5">
-                        <p>There is a CTF on every VM instance. <b><em>If</em></b>&nbsp;
-                        you manage to break out of the container, email
-                        <a href="#">to-be-defined@example.com</a>
-                        and you will be rewarded. <br/>
-                        <em>If</em>&nbsp; you bother this email address with anything
-                        that is not the ascii art contents of the flag file you will be
-                        ignored.</p>
-                    </div>
-                    <div class="col-md-2"></div>
-                    <div class="col-md-5">
-                        <p>Idea and implementation based on
-                        <a href="https://github.com/genuinetools/contained.af" target="_blank">github.com/genuinetools/contained.af</a>.</p>
-                    </div>
-                </div>
-            </footer>
-        </div>
-
-
-        <script src="js/contained.min.js"></script>
+        <form action="term.html">
+            Image Name: <input type = "text" name = "image" />
+            <br>
+            Contianer Port:
+            <select name="port">
+                <option style="display:none">
```

Comment:
Add `/` before the `>`

Reply:
What is `display:none` for?

Reply:
> What is `display:none` for?

When the first option is this, the dropdown has nothing selected by default, without this the first element in the dropdown is automatically selected. Allowing user to provide blank information.